--- highlights.lua	2025-03-19 15:59:25
+++ /Users/tormaale/git/bufferline.nvim/lua/bufferline/highlights.lua	2025-03-20 01:04:46
@@ -105,7 +105,7 @@
   if icon_hl_cache[icon_hl] then return icon_hl end
 
   local color_icons = config.options.color_icons
-  local color = not color_icons and parent.fg or nil
+  local color = not color_icons and "NONE"
   local hl_colors = vim.tbl_extend("force", parent, {
     fg = color or colors.get_color({ name = base_hl, attribute = "fg" }),
     ctermfg = color or colors.get_color({ name = base_hl, attribute = "fg", cterm = true }),
